docs/management_commands.rst

```diff
@@ -0,0 +1,74 @@
+Django Management Commands
+==========================
+
+Export Command
+--------------
+
+The ``export`` command allows you to export data from a specified Django model
+or a resource class. The exported data can be saved in different formats, such
+as CSV or XLSX.
+
+Usage
+-----
+
+.. code-block:: bash
+
+    python manage.py export <format> <resource> [--encoding ENCODING]
+
+- **format**: Specify the format in which the data should be exported. -
+- **resource**: Specify the resource or model to export. Accepts a resource class or a model class in dotted path format. - **--encoding** (optional): Specify the encoding (e.g., 'utf-8') to be used for the exported data.
+
+Example
+-------
+
+.. code-block:: bash
+
+    python manage.py export CSV auth.User
+
+This command will export the User model data in CSV format using utf-8
+encoding.
+
+Another example:
+
+.. code-block:: bash
+
+    python manage.py export XLSX mymodule.resources.MyResource
+
+This command will export the data from ``MyResource`` resource in XLSX format
+using utf-8 encoding.
+
+Import Command
+--------------
+
+The ``import`` command allows you to import data from a file into a specified
```

Comment:
I wonder if the language is slightly misleading... Would this be clearer:

> The ``import`` command allows you to import data from a file **using** a specified Django model or a custom resource class.

Reply:
in 0989f83